devicetypes/fibargroup/fibaro-dimmer-2-zw5.src/fibaro-dimmer-2-zw5.groovy

```diff
@@ -0,0 +1,498 @@
+/**
+ *  Fibaro Dimmer 2
+ */
+metadata {
+    definition (name: "Fibaro Dimmer 2 ZW5", namespace: "FibarGroup", author: "Fibar Group") {
+        capability "Switch"
+        capability "Switch Level"
+        capability "Energy Meter"
+        capability "Power Meter"
+        capability "Button"
+        capability "Configuration"
+        capability "Health Check"
+
+        command "reset"
+        command "refresh"
+        command "clearError"
+
+        attribute "errorMode", "string"
+
+        fingerprint mfr: "010F", prod: "0102", model: "2000"
+        fingerprint deviceId: "0x1101", inClusters:"0x5E,0x86,0x72,0x59,0x73,0x22,0x31,0x32,0x71,0x56,0x98,0x7A,0x20,0x5A,0x85,0x26,0x8E,0x60,0x70,0x75,0x27"

```

Comment:
this fingerprint should be removed because it only adds the security CC which is controlled either way.

To be honest, both of these command class-based fingerprints should be removed since the intention of this DTH is to be model-specific anyway.